.github/workflows/ci.yml

```diff
@@ -272,15 +272,14 @@ jobs:
           BABEL_ENV: test
           BABEL_8_BREAKING: true
           BABEL_TYPES_8_BREAKING: true
+      - name: Prepare artifacts
+        run: |
+          ./scripts/get-artifact-files.sh | tar --null -cvf babel-artifact.tar --files-from=-
```

Comment:
We don't need to compress the archive because `@actions/upload` will compress it during transfer.